### PR TITLE
feat: add item converter tool, link to minimessage viewer

### PR DESCRIPTION
**Pull request by MiniDigger**

### CODE REVIEW

File: docs/misc/tools/item-command-converter.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+slug: /tools/item-command-converter
+description: A tool to update commands from the old nbt based item format to item components
```
Comment (zlataovce) [inline docs/misc/tools/item-command-converter.mdx:3]:
```suggestion
description: A tool to update commands from the old NBT-based item format to item components.
```

File: docs/misc/tools/minimessage-web-editor.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+slug: /tools/minimessage-web-editor
+description: A web-based editor for creating and previewing MiniMessage-formatted text.
```
Comment (zlataovce) [inline docs/misc/tools/minimessage-web-editor.md:3]: the front matter title could exclude the heading's parentheses, looks a little nicer on the sidebar
```suggestion
title: MiniMessage Web Editor
description: A web-based editor for creating and previewing MiniMessage-formatted text.
```

File: docs/misc/tools/index.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import DocCardList from "@theme/DocCardList";
+import { useCurrentSidebarCategory } from "@docusaurus/theme-common";
+
+# Misc
```
Comment (Leguan16) [inline docs/misc/tools/index.mdx:4]:
```suggestion
# Tools
```

**Review by Leguan16 — COMMENTED** [review]
Maybe move the tools section all the way down to the bottom?

**Comment by olijeffers0n** [comment]
> Maybe move the tools section all the way down to the bottom?

It is standard for us to put categories at the top

**Review by kennytv — CHANGES_REQUESTED** [review]
Just to make sure this isn't merged early: Needs to wait on more backend changes + it should get a dropdown for switching from the default `Command` to `Item argument` and `Component argument` with their different endpoints

File: src/components/ItemCommandConverter.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    await navigator.clipboard.writeText(output);
+    toggleState(setCopySuccess);
+  };
+
+  return (
+    <div className="item-command-converter">
+      <label>
+        Input:
+        <textarea
+          placeholder="Enter your pre 1.20.5 command here..."
```
Comment (Machine-Maker) [inline src/components/ItemCommandConverter.tsx:54]: Should specify exactly 1.20.4. Afaik, the convert is only converting from 1.20.4 to 1.20.5, not < 1.20.5 to 1.20.5

**Review by Machine-Maker — CHANGES_REQUESTED** [review]

**Review by olijeffers0n — APPROVED** [review]

**Pull request merged**